tests/contrib/django/django_app/urls.py

```diff
@@ -13,6 +13,12 @@
 from .. import views
 
 
+if django.VERSION < (2, 0, 0):
```

Comment:
this shouldn't be used with django 1 right, so is this needed?

Reply:
Nope but this was a mistake that line should be `django.VERSION < (3, 0, 0)`. I'd like to keep test coverage for  django.conf.urls.url for django 2